libs/common/src/lib/decorators/transform-boolean-string.decorator.ts

```diff
@@ -0,0 +1,23 @@
+import { IsBoolean, IsOptional } from "class-validator";
+import { Transform } from "class-transformer";
+import { applyDecorators } from "@nestjs/common";
+
+type BooleanStreamOptions = {
```

Comment:
This is my mistake, but please rename this to `BooleanStringOptions`.

Reply:
changed